R/add_stat.R

```diff
@@ -148,6 +148,8 @@ add_stat <- function(x, fns, location = everything() ~ "label") {
     dplyr::tibble(variable = names(fns)) |>
     dplyr::mutate(
       summary_type = map_chr(.data$variable, ~ x$inputs$type[[.x]]),
+      # tbl_continuous objects generally didn't have a "$inputs$type" object nested inside
```

Comment:
explanation is here, `tbl_continuous` objects didn't have `x$inputs$type` inside them